wolox-books/src/app/screens/auth/screens/login/components/login-form/login-form.component.html

```diff
@@ -1,9 +1,15 @@
 <app-auth-form [secondaryButtonText]="secondaryButtonText" [secondaryButtonRedirectionUrl]="secondaryButtonRedirectionUrl">
-	<form [formGroup]>
-		<app-text-input [inputName]="'name'" [inputLabel]="'Name'" [inputType]="'text'"></app-text-input>
-		<app-text-input [inputName]="'password'" [inputLabel]="'Password'" [inputType]="'password'"></app-text-input>
+	<form [formGroup]="loginForm" (ngSubmit)="onSubmit(loginForm.value)">
+		<app-text-input [inputName]="'email'" [inputLabel]="'Email'" [inputType]="'text'" [parentForm]="loginForm"></app-text-input>
```

Comment:
you can avoid the "[" on the inputs if you are going to use strings, same with the single quotes